### PR TITLE
[WPE][GTK] New section about Multimedia

### PR DESCRIPTION
**Pull request by philn**
Migration of the https://trac.webkit.org/wiki/WebKitGTK/Debugging#Debuggingmultimediastuff section.

### CODE REVIEW

File: docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+located in a folder accessible in read-write mode by the Web content process,
+using the `WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS` environment variable and
+the `--filesystem=home` flatpak option.
+
+
+```shell
+mkdir -p $HOME/dots
+flatpak run --filesystem=home --env="WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS=1" \
+    --env="GST_DEBUG=3,webkit*:6" --env="GST_DEBUG_FILE=$HOME/gst.log"
+    --env="GST_DEBUG_DUMP_DOT_DIR=$HOME/dots org.gnome.Epiphany -p "https://..."
```
Comment (mcatanzaro) [inline docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:41]:
```suggestion
    --env="GST_DEBUG_DUMP_DOT_DIR=$HOME/dots" org.gnome.Epiphany -p "https://..."
```

**Review by mcatanzaro — APPROVED** [review]

File: docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Once gathered, the log file and pipeline graph dumps can be zipped together and
+uploaded online. Assuming the commands are executed as shown in the next
+sections, the log file will be `$HOME/gst.log` and the pipeline graph dumps will
+be present in the `$HOME/dots` folder.
+
+## Flatpak apps
+
+In this section we take the example of the [GNOME
+Web](https://flathub.org/apps/org.gnome.Epiphany) app, a.k.a Epiphany.
```
Comment (mcatanzaro) [inline docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:28]:
```suggestion
Web](https://flathub.org/apps/org.gnome.Epiphany) app, a.k.a. Epiphany.
```

File: docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+```shell
+mkdir -p $HOME/dots
+flatpak run --filesystem=home --env="WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS=1" \
+    --env="GST_DEBUG=3,webkit*:6" --env="GST_DEBUG_FILE=$HOME/gst.log"
+    --env="GST_DEBUG_DUMP_DOT_DIR=$HOME/dots org.gnome.Epiphany -p "https://..."
+```
+
+GNOME Web has [three different
+flavours](https://gitlab.gnome.org/GNOME/epiphany#download-and-install). The
+command above is for the stable version (`org.gnome.Epiphany`). The TechPreview
```
Comment (mcatanzaro) [inline docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:46]:
```suggestion
command above is for the stable version (`org.gnome.Epiphany`). The Tech Preview
```

File: docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+application name is `org.gnome.Epiphany.Devel` and the Canary version is called
+`org.gnome.Epiphany.Canary`. So depending on which version you test, the
+command-line will need to be adapted accordingly.
```
Comment (mcatanzaro) [inline docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:49]:
```suggestion
command line will need to be adapted accordingly.
```

**Review by mcatanzaro — COMMENTED** [review]
Thanks for adding this important page.

Unimportant nits:

File: docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md (hunk 1 above)
Comment (mcatanzaro) [reply docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:41]: Ah, we also need `GST_DEBUG_NO_COLOR=1`

Comment (philn) [reply docs/Ports/WebKitGTK and WPE WebKit/Multimedia.md:41]: well, that's not mandatory... `less -R` can show the colors fine.

**Pull request merged**